README.md

```diff
@@ -27,14 +27,32 @@ Feel free to contribute or start using the problems!
 
 ## How to build
 
-- The subprojects are currently in subdirectories in `theories`, roughly corresponding to papers or theses covering them.
-- External libraries are in the `external` directory
-- `git submodule init && git submodule update` initialises the external libraries
-- You need to install the [Equations](https://mattam82.github.io/Coq-Equations/) package for Coq, see the page for detailed instructions
-- `make all` builds all subprojects by calling `make all` of the corresponding subproject's makefile
+### Required packages
+
+You need `Coq 8.8.1` or `8.8.2` built on OCAML `>= 4.02.3` and the [Equations](https://mattam82.github.io/Coq-Equations/) package for Coq. If you're using opam 2 you can use the following commands to install the dependencies on a new switch:
```

Comment:
**`OCaml 4.02.3` did not work for me**. I only tested `OCaml 4.07.0` successfully

Reply:
You're right, fixed (also in the `opam` file)